tl/cli/context_match.py

```diff
@@ -0,0 +1,41 @@
+import typing
+import argparse
+import sys
+import traceback
+import tl.exceptions
+
+
+def parser():
+    return {
+        'help': 'Match the context values to the properties and add the score of the match.'
+    }
+
+
+def add_arguments(parser):
+    from tl.utility.utility import Utility
+    from kgtk.cli.text_embedding import ALL_EMBEDDING_MODELS_NAMES
+    # input file
+    parser.add_argument('input_file', nargs='?', type=argparse.FileType('r'), default=sys.stdin)
+    # debug
+    parser.add_argument('context_file', nargs='?', type=argparse.FileType('r'), default=sys.stdin)

```

Comment:
change the input parameter to `--context-file`. This should be of type `str` and should be a file path. Not stdin

Reply:
Done.